src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```diff
@@ -25,7 +25,7 @@
     "required": "*",
     "requiredFields": "Die mit einem Stern (*) markierten Felder sind Pflichtfelder.",
     "privacyTitle": "Datenschutz",
-    "privacyNotice": "Ich habe die <a data-toggle=\"modal\" data-bs-toggle=\"modal\" data-url=\"%privacyUrl%\" href=\"%privacyUrl%\" title=\"Datenschutzbestimmungen\">Datenschutzbestimmungen</a> zur Kenntnis genommen und die <a data-toggle=\"modal\" data-bs-toggle=\"modal\" data-url=\"%tosUrl%\" href=\"%tosUrl%\" title=\"AGB\">AGB</a> gelesen und bin mit ihnen einverstanden.",
+    "privacyNotice": "Ich habe die <a data-toggle=\"modal\" data-bs-toggle=\"modal\" data-url=\"%privacyUrl%\" href=\"%privacyUrl%\" title=\"Datenschutzbestimmungen\">Datenschutzbestimmungen</a> zur Kenntnis genommen und erkenne diese an.",
```

Comment:
This will change the privacy notice everywhere even for checkouts etc. which is not what is wanted right?

Instead please create a new snippet each inside the `contact` and `newsletter` sections instead of the `general` section here, and then use those specific snippets in the places where the TOS is not required

This also has the benefit that the snippets could be adjusted independently from each other if that is needed because of further law changes or because of details in specific projects